cypress-prioritization-and-triage.md

```diff
@@ -0,0 +1,139 @@
+# The Cypress App Prioritization and Triage Methodology
+
+At Cypress, we love our open source community. We work every day to grow our partnership with the community through open, honest, and straightforward communication about our processes and plans for the platform. The goal of this document is to provide resources that show our plans for the future and serve as a guide to our processes around handling issues that are raised by the open source community.
```

Comment:
```suggestion
At Cypress, we love our open source community. We work every day to grow our partnership with the community through open, honest, and straightforward communication about our processes and plans for the platform. The goal of this document is to provide resources that show our plans for the future and serve as a guide for our processes around handling issues that are raised by the open source community.
```

Reply:
I did some research on this topic and have found that a guide to is in fact correct in this usage.